gnat2goto/driver/gnat2goto_itypes.adb

```diff
@@ -190,21 +190,39 @@ package body Gnat2goto_Itypes is
          when N_Integer_Literal => Lower_Bound_Value :=
               Store_Nat_Bound (Bound_Type_Nat (Intval (Lower_Bound)));
          when N_Identifier => Lower_Bound_Value :=
-              Store_Symbol_Bound (Bound_Type_Symbol (Lower_Bound));
+              Store_Symbol_Bound (Bound_Type_Symbol (Do_Identifier
```

Comment:
This should probably just be `Lower_Bound_Value := <case expression>`, with the unhandled case returning whatever the default value here is (I notice theere doesn't seem to be a default value right now which is UB even in Ada)